samples/src/Microsoft.Azure.IIoT.App/src/Models/PublisherInfoRequested.cs

```diff
@@ -9,43 +9,32 @@ namespace Microsoft.Azure.IIoT.App.Models {
     public class PublisherInfoRequested {
 
         public PublisherInfoRequested(PublisherInfo publisher) {
-            _requestedMaxWorkers = publisher?.PublisherModel?.Configuration?.MaxWorkers;
-            _requestedHeartbeatInterval = publisher?.PublisherModel?.Configuration?.HeartbeatInterval;
-            _requestedJobCheckInterval = publisher?.PublisherModel?.Configuration?.JobCheckInterval;
-        }
+            RequestedMaxWorkers = publisher?.PublisherModel?.Configuration?.MaxWorkers.ToString();
+
+            TimeSpan? heartbeatInterval = publisher?.PublisherModel?.Configuration?.HeartbeatInterval;
+            if (heartbeatInterval.HasValue && heartbeatInterval >= TimeSpan.Zero) {
```

Comment:
heartbeatInterval could be null

Reply:
See previous comment on jobCheckInterval.

Reply:
👌